src/app/js/public/index.js

```diff
@@ -30,16 +31,24 @@ const store = configureStore(
 );
 
 render(
-    <Root store={store} routes={<Routes />} history={history} />,
+    <Provider {...{ store }}>
+        <MuiThemeProvider>
+            <Routes history={history} />
+        </MuiThemeProvider>
+    </Provider>,
     document.getElementById('root'),
 );
 
 // Hot Module Replacement API
 if (module.hot) {
     module.hot.accept('../Root', () => {
-        const NewRoot = require('../Root').default; // eslint-disable-line
+        const NewRoutes = require('./Routes').default; // eslint-disable-line
```

Comment:
Are you sure that we still need this `module.hot` condition now we have a webpack dev server?